src/js/selection/SelectionManager.js

```diff
@@ -91,7 +93,29 @@
     }
   };
 
-  ns.SelectionManager.prototype.paste = function() {
+  // Iterate through pixels to get the largest x and y values of a selection.
+  ns.SelectionManager.prototype._getBoundaryEdges = function(pixels) {
```

Comment:
Also, this method is effectively static.  It'd be nice to write a unit test for it.